### ISSUE TITLE
Provide the ability to access 'name' or 'text content' on a Draggable for better screen reader experience

### ISSUE THREAD

**Issue by nramamurth** [atlassian/react-beautiful-dnd#2341]
### Description

I have an unordered list of list item elements (that are `Draggable`s). I'd like the screen reader to announce the visible content of a particular list item when the item is lifted using Spacebar. The `draggableId` cannot be used for our use-case and the [Use Names Where Possible](https://github.com/atlassian/react-beautiful-dnd/blob/master/docs/guides/screen-reader.md#use-names-where-possible) section mentions about providing the ability for us to maybe add the content as a prop to the `Draggable` itself so it is accessible in the `onDragStart` handler for the announcer to take care of our custom messages. 

Is there a way for me to achieve what I want to with the current latest version of the library or is this something that can be be supported in an upcoming release?
